src/components/MediaBrowser/MediaBrowserData.js

```diff
@@ -61,11 +61,10 @@ function MediaBrowserData({ docType }) {
     isInitialLoading,
   } = useInfiniteQuery(
```

Comment:
Not essential to do now, but we will want to move this into a dataHook, similar to useSearchLoader. Could be moved we re-implement  media search.

Reply:
Sounds good. I'll add the note to the media-search ticket.